book/src/configuration.md

```diff
@@ -89,7 +89,7 @@ The `[editor.statusline]` key takes the following sub-keys:
 
 | Key           | Description | Default |
 | ---           | ---         | ---     |
-| `left`        | A list of elements aligned to the left of the statusline | `["mode", "spinner", "file-name", "file-modification-indicator"]` |
+| `left`        | A list of elements aligned to the left of the statusline | `["mode", "spinner", "file-name", "file-modification-indicator", "read-only-indicator"]` |
```

Comment:
The read-only and modification indicators should be swapped here to match the `Default for StatusLineConfig`